docs/preview/02-Features/endpoint-validation.md

```diff
@@ -18,6 +18,8 @@ The features described here require the following package:
 PM> Install-Package Arcus.EventGrid.Security.WebApi
 ```
 
+> ⚠ This package used to be called `Arcus.EventGrid.Security`. Please make sure that you migrate towards this package as it's not being actively maintained beyond v3.2.
```

Comment:
Should we put this before the Install-Package command ?
It should definitely stand out that this is deprecated; could be put in bold ?

Reply:
We can do both 😉 .

Reply:
Done that.